src/commands/timelinecommands.cpp

```diff
@@ -492,18 +492,18 @@ void MoveClipCommand::redo()
         }
     }
     // First, remove each clip.
+    int i = 0;
     for (auto &clip : m_selection) {
         if (!m_redo) {
             // On the initial pass, remove clips while recording info about them.
             QScopedPointer<Mlt::ClipInfo> info(m_model.findClipByUuid(MLT.uuid(clip), trackIndex, clipIndex));
             if (info && info->producer && info->producer->is_valid() && info->cut) {
-                info->producer->set(kNewTrackIndexProperty, qBound(0, trackIndex + m_trackDelta,
-                                                                   m_model.trackList().size() - 1));
-                info->producer->pass_property(*info->cut, kPlaylistStartProperty);
```

Comment:
Question for @ddennedy:

I see the following properties being used in other parts of the code:
* kPlaylistStartProperty
* kTrackIndexProperty

I do not fully understand the purpose of those producer properties. Should they continue to be set here since they seem to be accessed in other parts of the code?

Reply:
Yes these must be updated when a clip moves.

Reply:
I restored lines that set values to these properties.
I think here is fine instead of later.

Reply:
I just checked about `kTrackIndexProperty`.

When a user moves a clip from V2 (track index 0) to V1 (track index 1), kTrackIndexProperty remains 0 because it only registers a value retrieved from `m_model.findClipByUuid`, not a value calculated by `trackIndex + m_trackDelta`.
`kTrackIndexProperty` is relatively new entry in `shotcut_mlt_properties.h`, so the only affected place is `glaxnimateproducerwidget.cpp`

I think this is a separate issue. I haven't even looked into the issue of `kPlaylistStartProperty` and `UndoHelper`. Maybe this one is fine since it's been around like 5 years. We can merge this pull request after additional critiques if there is any.

Reply:
I do not have any further comments, but I would like to see a thumbs up from Dan before we resolve this comment.